modular_skyrat/modules/chaplain/code/chaplainmodule.dm

```diff
@@ -0,0 +1,23 @@
+// FILE FOR CHAPLAIN RELATED SKYRAT CODE
+
+// Nun outfit
+/obj/item/clothing/head/nun
+	name = "nun headdress"
+	desc = "A cloth headpiece that covers the hair and ears. The iconic uniform of nuns across the galaxy."
+	icon = 'modular_skyrat/modules/chaplain/icons/nun.dmi'
+	worn_icon = 'modular_skyrat/modules/chaplain/icons/nunmob.dmi'
+	icon_state = "nun_head"
+	inhand_icon_state = "cowboy_hat_black"
+	flags_inv = HIDEHAIR | HIDEEARS
+
+/obj/item/clothing/under/nunrobes
+	name = "nun robes"
+	desc = "Plain black robes in a feminine style, meant to display piety and turn thoughts away from sin."
+	icon = 'modular_skyrat/modules/chaplain/icons/nun.dmi'
+	worn_icon = 'modular_skyrat/modules/chaplain/icons/nunmob.dmi'
```

Comment:
This one in particular should be put into the 'civilian' icon file in aforementioned master files, OR made into a suit item - Wearing the robes over the uniform tends to be the choice for chaplain outfits, especially when they're as long as these ones.